### ISSUE TITLE
Missing license & attribution for Moka assets.

### ISSUE THREAD

**Issue by snwh** [linuxmint/mint-y-icons#1]
Some Moka users brought this theme to my attention, especially that the [CC-BY-SA 4.0](https://github.com/moka-project/moka-icon-theme/blob/master/LICENSE_CCBYSA) license of the Moka assets that are being reused is being ignored as well the credit to the original author (which is me) is absent.

I'd thank you to fix this, or find another theme to base this icon set on.

**Comment by clefebvre** [comment]
Hi Sam,

Sorry about this. We just got started with this and we're still experimenting. It's very likely we'll base on Moka going forward but nothing is set in stone yet.

In any case, we do intend on giving full public credit to original authors/projects in the theme itself and in our communication and we'll respect the licensing of course.

We also want to get in touch with people whose work we're reusing. First to let them know what we're doing, to thank them, to send them a donation (that's something we usually do, especially for artwork) and to see if they're interested in working with us or not.

**Comment by aleksandar-todorovic** [comment]
@clefebvre Sorry, but I fail to realize how is "we've just started working on this" any kind of excuse for doing something like this.

If you've copied some of the icons from the Moka project, you should have at least stated in your README file that you did so.

And what's even worse than not giving the appropriate credit back to Sam, is that, by not using the same license, you're breaking the SA part of CC-BY-SA license. By not choosing any license, you're [defaulting to](http://choosealicense.com/no-license/) All Rights Reserved, and therefore literally stealing the work done by someone else (in this case Sam) and claiming it to be your own.

This is not OK. "We just got started" is not a good excuse for doing something like this.

**Comment by embik** [comment]
There *is* a license for this repository hidden away in [debian/copyright](https://github.com/linuxmint/mint-y-icons/blob/master/debian/copyright) which is GPL3. Make of that what you will.

Moreover, that file is claiming copyright of @clefebvre for *all* files (`*`). That's most likely result of "still experimenting" and quickly setting up a `debian/` directory, but it's still a false claim and makes me worry about Linux MInt's dedication to value and comply with FOSS licensing. Breach of license seems to be shrugged off although this is a public repository and therefore of course subject to licensing terms.

**Comment by clefebvre** [comment]
The technical argument is sound and we all want to see it fixed. And we will get to that.

What I find a bit weird here, is how you phrase this, so let me reassure you on the intent here:

- We are fixing this.
- There is no agenda to steal or claim other people's work as being our own.

These are uncalled for:

"makes me worry about Linux MInt's dedication to value and comply with FOSS licensing"
"therefore literally stealing the work done by someone else (in this case Sam) and claiming it to be your own"

It's like you guys already got pumped up about all this already? Is there a controversial thread or reddit we're not aware of?

Sam created an issue, I got wind of it this morning, this is getting fixed today. I'm not sure what the story is, but I really don't see a point not to sound friendly on your first communication with us.

**Comment by snwh** [comment]
@clefebvre thanks, but I think some folks see this a moral argument more than a technical one given what FOSS licenses represent, myself included, but I try to refrain from being sensationalist.

Experimenting with icons fine, but you all did **publicly** publish the source for my icons under a different license which is problematic and people did catch wind of. In my view, that is a bit of carelessness from what is a pretty prominent project.

**Comment by embik** [comment]
@clefebvre The story is that you have had a repository public *for a month* containing obvious copyright and licensing violations. Complying with licenses seem to be the first thing for a project containing upstream work to me. The fact that the original author had to point out your violations to get it fixed makes me worry about how much Linux Mint values copyright and licensing in their **public** codebase. That's all I'm saying.

**Comment by clefebvre** [comment]
Point taken. 

Sam, can you check https://github.com/linuxmint/mint-y-icons/commit/262c757d0ea6730b156edda39373bb390479dddd and tell me if anything needs editing or adding?

**Comment by snwh** [comment]
@clefebvre thanks

**Comment by clefebvre** [comment]
No problem, I'm glad we got it sorted. We'll review some of the other temporary projects (there's a draft of GTK theme based on Arc) we started and add licensing there as well.

I'd like to get in touch with you Sam also. First because we want to thank you and donate to you for the work on Moka, and second because we're interested in working with you going forward, whether it's to potentially sync changes from/to Moka or contract the production of new icons.

It's a bit early still, and I thought we'd get in touch in about a month time, at a time when we were sure we were sticking to Moka and we had a better idea of where we were going with this. But heh, at least you know we're here, we love what you've done and we can't wait to see how we can use it.

If you ever need to quickly get in touch with us (note how this took 3 days to get to me via github) we're on the IRC #linuxmint-dev on irc.spotchat.org.
